src/features/runtime-checks.js

```diff
@@ -471,6 +470,21 @@ function overrideCreateElement () {
     initialCreateElement = proxy._native
 }
 
+function getTaintFromScope (scope, args) {
+    try {
+        scope = args.callee.caller
+    } catch {
+        console.log('taint: failed to get callers scope', args, scope)
+    }
+    return hasTaintedMethod(scope)
+}
+
+function injectGenericOverloads () {
```

Comment:
Assuming you intend to follow up on this in a subsequent PR?

Reply:
Yeah testing that on real sites now to see the impacts.